wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -600,6 +602,12 @@ private void check() {
 
                 log.info("starting peergroup");
                 peerGroup = new PeerGroup(Constants.NETWORK_PARAMETERS, blockChain, headerChain);
+                if (resetMNListsOnPeerGroupStart) {
+                    resetMNListsOnPeerGroupStart = false;
+                    application.getWallet().getContext().masternodeListManager.setBootstrap(mnlistinfoBootStrapStream, qrinfoBootStrapStream, SimplifiedMasternodeListManager.QUORUM_ROTATION_FORMAT_VERSION);
```

Comment:
I don't like this function call.  Some DashJ objects are accessible through a DashJ `Context`.  There is probably a better way to organize this in DashJ.